requirements.txt

```diff
@@ -1,9 +1,9 @@
 PyYAML==3.11
 cached-property==1.2.0
-docker-py==1.8.1
 dockerpty==0.4.1
 docopt==0.6.1
 enum34==1.0.4
+git+https://github.com/docker/docker-py.git@26f2b696ab61a6dd41a19c9730f85a6b572089b7#egg=docker-py
 jsonschema==2.5.1
```

Comment:
Hi there, i'm not really sure that we should do something like this, can't we wait until docker-py will support internal networks (until new version will see the light)?